### PR TITLE
Created all tasks

### PR DESCRIPTION
**Pull request by manthanand**

### CODE REVIEW

File: Core/Inc/main.h [hunk 1]
```diff
@@ -29,10 +29,9 @@ extern "C" {
 
 /* Includes ------------------------------------------------------------------*/
 #include "stm32f4xx_hal.h"
-
 /* Private includes ----------------------------------------------------------*/
 /* USER CODE BEGIN Includes */
-
+#include "Tasks.h"
```
Comment (ClarkPoon) [inline Core/Inc/main.h]: Why is this included here? It doesn't look like you are using it in this file

Comment (manthanand) [reply Core/Inc/main.h]: It is needed to include the functions for each of the tasks when I call `osThreadNew`

File: Core/Src/main.c [hunk 2]
```diff
@@ -145,9 +163,10 @@ int main(void)
   /* Create the thread(s) */
   /* creation of defaultTask */
   defaultTaskHandle = osThreadNew(StartDefaultTask, NULL, &defaultTask_attributes);
-
   /* USER CODE BEGIN RTOS_THREADS */
-  /* add threads, ... */
+  DataReadingTaskHandle = osThreadNew(DataReadingTask, NULL, &DataReadingTask_attributes);
+  DataLoggingTaskHandle = osThreadNew(DataLoggingTask, NULL, &DataLoggingTask_attributes);
+  BroadcastingTaskHandle = osThreadNew(BroadcastingTask, NULL, &BroadcastingTask_attributes);
```
Comment (manthanand) [inline Core/Src/main.c:170]: Right here.

**Review by ClarkPoon — APPROVED** [review]
